pkgs/development/python-modules/apache-airflow/pendulum-fix.patch

```diff
@@ -0,0 +1,41 @@
+diff --git a/airflow/settings.py b/airflow/settings.py
```

Comment:
can this be converted to a PR upstream (if it isn't already), and `fetchpatch`-ed from there?

Reply:
There was one that was stalled: https://github.com/apache/airflow/pull/4507

I will create a new one.

Reply:
Created: https://github.com/apache/airflow/pull/6561#issuecomment-553072328